scripts/openstack-quickstart-demosetup

```diff
@@ -506,6 +506,20 @@ if [ "x$with_tempest" = "xyes" ]; then
     crudini --set /etc/neutron/neutron_lbaas.conf service_providers service_provider "LOADBALANCERV2:Haproxy:neutron_lbaas.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver:default"
 fi
 
+# NOTE(toabctl): epmd only listens on the loopback interface which does not work
+mkdir -p /etc/systemd/system/epmd.socket.d
+cat <<EOF > /etc/systemd/system/epmd.socket.d/ports.conf
+[Socket]
+ListenStream=
+ListenStream=[::]:4369
+EOF
+systemctl daemon-reload
+systemctl restart epmd.socket epmd.service
+
+# as workaround, we need to install rabbitmq-server-plugins to not get:
+# {"init terminating in do_boot",{undef,[{rabbit_nodes,ensure_epmd,[],[]},{rabbit_cli,start_distribution,0,[{file,"src/rabbit_cli.erl...
```

Comment:
@dirkmueller are you fine with merging this as workaround until we have the fix in rabbitmq-server ?